bansystem-lib/src/main/java/ch/dkrieger/bansystem/lib/command/defaults/KickCommand.java

```diff
@@ -43,6 +43,12 @@ public KickCommand() {
     }
     @Override
     public void onExecute(NetworkCommandSender sender, String[] args) {
+        if(!sender.hasPermission(reason.getPermission())&& !sender.hasPermission("dkbans.*")){
+            sender.sendMessage(Messages.REASON_NO_PERMISSION
+                    .replace("[prefix]",getPrefix())
+                    .replace("[reason]",reason.getDisplay()));
+            return;
+        }
```

Comment:
"reason" variable is not available at this point. Which permission is not working?
Permission is already checked in command manager.

Reply:
You can try normal user can use kick and mute command.